src/main/java/core/basesyntax/FieldXY.java

```diff
@@ -13,7 +13,98 @@
 public class FieldXY {
 
     public void moveRobot(Robot robot, int toX, int toY) {
-        //your code
+        if (robot.getX() < toX && robot.getY() < toY) {
+            setDirection(robot, Direction.UP);
```

Comment:
Good, but actually you might make this solution shorter, FYI:

  if (robot.getX() < toX) {
            setDirection(robot, Direction.RIGHT);
        } else {
            setDirection(robot, Direction.LEFT);
        }

        while (robot.getX() != toX) {
            robot.stepForward();
        }

        if (robot.getY() < toY) {
            setDirection(robot, Direction.UP);
        } else {
            setDirection(robot, Direction.DOWN);
        }

        while (robot.getY() != toY) {
            robot.stepForward();
        }